blivet/udev.py

```diff
@@ -322,6 +322,31 @@ def device_is_loop(info):
             os.path.isdir("%s/loop" % device_get_sysfs_path(info)))
 
 
+@util.deprecated("3.0", "")
```

Comment:
This should suggest a replacement for the user code or explain why this function is no longer needed.